app/boards/shields/fourier/fourier.dtsi

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2021 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#include <dt-bindings/zmk/matrix_transform.h>
+
+/ {
+	chosen {
+	    zmk,kscan = &kscan0;
+	    zmk,matrix_transform = &default_transform;
+	};
+
+	/*
+	 * This transform correspondsto the 60% left without macro keypad and 65% right, even this
+	 * combination of PCBs can have keys in different locations based on configuration.
+	 */
+	default_transform: keymap_transform_0 {
+		compatible = "zmk,matrix-transform";
+		columns = <13>;
+		rows = <4>;
+		map = <
+RC(0,0) RC(0,1) RC(0,2) RC(0,3) RC(0,4) RC(0,5)      /**/  RC(0,6) RC(0,7) RC(0,8) RC(0,9) RC(0,10) RC(0, 11) RC(0,12)
+RC(1,0)   RC(1,1) RC(1,2) RC(1,3) RC(1,4) RC(1,5)    /**/  RC(1,6) RC(1,7) RC(1,8) RC(1,9) RC(1,10) RC(1,12)
+RC(2,0)     RC(2,1) RC(2,2) RC(2,3) RC(2,4) RC(2,5)  /**/    RC(2,6) RC(2,8) RC(2,9) RC(2,10) RC(2,11) RC(2,12)
+RC(3,0)   RC(3,1) RC(3,2) RC(3,3) RC(3,4)            /**/  RC(3,6)        RC(3,9)  RC(3,10)  RC(3,11)  RC(3,12)
+		>;
+	};
+
+	kscan0: kscan {
+		compatible = "zmk,kscan-gpio-matrix";
+		label = "KSCAN";
+
+		diode-direction = "col2row";
+		row-gpios
+            = <&pro_micro 21 (GPIO_ACTIVE_HIGH | GPIO_PULL_DOWN)>  // Row A
+            , <&pro_micro 6 (GPIO_ACTIVE_HIGH | GPIO_PULL_DOWN)>   // Row B
+            , <&pro_micro 7 (GPIO_ACTIVE_HIGH | GPIO_PULL_DOWN)>   // Row C
+            , <&pro_micro 8 (GPIO_ACTIVE_HIGH | GPIO_PULL_DOWN)>   // Row D
+            ;
```

Comment:
```suggestion
			= <&pro_micro 21 (GPIO_ACTIVE_HIGH | GPIO_PULL_DOWN)>  // Row A
			, <&pro_micro 6 (GPIO_ACTIVE_HIGH | GPIO_PULL_DOWN)>   // Row B
			, <&pro_micro 7 (GPIO_ACTIVE_HIGH | GPIO_PULL_DOWN)>   // Row C
			, <&pro_micro 8 (GPIO_ACTIVE_HIGH | GPIO_PULL_DOWN)>   // Row D
			;
```
Switched to tabs for consistent spacing in the file

Reply:
Fixed these and two others I found in that file